src/Blocks/continuous.jl

```diff
@@ -417,16 +417,28 @@ function LimPID(; name, k=1, Ti=false, Td=false, wp=1, wd=1,
 end
 
 """
-    StateSpace(A, B, C, D=0; x_start=zeros(size(A,1)), name)
+    StateSpace(A, B, C, D=0; x_start=zeros(size(A,1)), u0=zeros(size(B,2)), y0=zeros(size(C,1)), name)
 
 A linear, time-invariant state-space system on the form.
-```
+```math
 ẋ = Ax + Bu
 y = Cx + Du
 ```
 Transfer functions can also be simulated by converting them to a StateSpace form.
+
+`y0` and `u0` can be used to set an operating point, providing them changes the dynamics to
```

Comment:
```suggestion
`y0` and `u0` can be used to set an operating point, providing them changes the dynamics from an LTI system to the affine system 
```

Reply:
@ValentinKaisermayer is this a good clarification?

Reply:
Looks good!